docs/how-to/manage-domains.md

```diff
@@ -0,0 +1,12 @@
+# How to manage domains
+
+To manage the list of domains a user is allowed to request changes to, the following actions are available
+
+## Allowing domains
+Run `juju run --wait=5s httprequest-lego-provider/0 allow-domains username=example domains="example.domain.com,example2.domain.com" to add domains to the list of allowed domains.
```

Comment:
I'd separate the juju run command onto its own line for clarity. So maybe "To add domains to the list of allowed domains for a particular user, run:" and then the command on a separate line. Same for others below.